mmv1/third_party/tgc/getconfig.go

```diff
@@ -32,8 +32,8 @@ func NewConfig(ctx context.Context, project, zone, region string, offline bool,
 		"GOOGLE_IMPERSONATE_SERVICE_ACCOUNT",
 	})
 
+	transport_tpg.ConfigureBasePaths(cfg)
```

Comment:
this seems fine; could you elaborate on why it's necessary?

Reply:
For test case `example_compute_global_forwarding_rule`, the backend service field uses the baes path: 
https://github.com/GoogleCloudPlatform/terraform-google-conversion/blob/3703a3ab8bbb7178757d1b48b01d0574b7cb8ad6/tfplan2cai/converters/google/resources/services/compute/compute_forwarding_rule.go#L254